CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 - Add support for using `Config`, `getProject()`, `getStack()`, and `isDryRun()` from Policy Packs
   via upgraded dependency on `@pulumi/pulumi` v1.8.0 (requires v1.8.0 of the Pulumi SDK).
 
+- Provide easier type checking for `validateStack` (https://github.com/pulumi/pulumi-policy/pull/173).
```

Comment:
This is fine for now, but post our 1.0 launch we might want to have code snippets / examples in the PR's description so someone looking at the change log can find out more, etc. But while we are still feeling things out, the level of detail you have in this PR's description now is just fine.